packages/oui-numeric/src/numeric.controller.js

```diff
@@ -0,0 +1,117 @@
+import _ from 'lodash'
+
+// By design, value is restricted to [0, 99999] interval
+const MIN_VALUE = 0
+const MAX_VALUE = 99999
+
+export default class {
+  constructor ($attrs, $element, $log) {
+    'ngInject'
+
+    this.$attrs = $attrs
+    this.$element = $element
+    this.$log = $log
+  }
+
+  $postLink () {
+    this.$element.removeAttr('id')
+    this.$element.removeAttr('name')
+  }
+
+  $onInit () {
+    if (!this.id) {
+      this.$log.warn(`Missing required attribute id`)
+    }
+
+    if (!this.text) {
```

Comment:
Angular is not already throwing an error if `text` is missing because you are using `@` in bindings ?

Reply:
No it doesn't, value will be undefined